app/src/main/java/com/thumbjive/atennapod/VoiceControlService.java

```diff
@@ -0,0 +1,205 @@
+package com.thumbjive.atennapod;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.File;
+import java.io.IOException;
+
+import de.danoeh.antennapod.activity.MainActivity;
+import de.danoeh.antennapod.core.service.playback.PlayerStatus;
+import de.danoeh.antennapod.core.util.playback.PlaybackController;
+import edu.cmu.pocketsphinx.Assets;
+import edu.cmu.pocketsphinx.Hypothesis;
+import edu.cmu.pocketsphinx.RecognitionListener;
+import edu.cmu.pocketsphinx.SpeechRecognizer;
+import edu.cmu.pocketsphinx.SpeechRecognizerSetup;
+
+/**
+ * Experimental support for play/stop voice commands
+ *
+ * Derived from:
+ *   https://github.com/cmusphinx/pocketsphinx-android-demo
+ */
+
+public class VoiceControlService
+        implements RecognitionListener {
+
+    private SpeechRecognizer recognizer;
+    private static final String KWS_SEARCH = "wakeup";
+    private static final String KEYPHRASE = "wakeup";
```

Comment:
The first should really probably be an enum.  It's an arbitrary key to the current recognizer vocabulary to use.  The second is the single word to recognize when in the "wakeup" mode. (This pattern was copied from the reference app, but the key phrase was "Oh mighty computer").